docs/landing/operations/deployment/heroku.md

```diff
@@ -0,0 +1,126 @@
+# Deploy on Heroku
+
+To deploy your app to Heroku, you need to follow a few simple steps and you are ready to go.
+
+This guide assumes you have
+
+ * [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli) installed and configured
+ * created a working local project with `shopware-pwa init`
+ * created a new application in your Heroku dashboard
+
+## Heroku Setup
+
+According to the Nuxt.js documentation on deploying to Heroku, you have to set two environment variables in your Heroku App. You can set those in your Applications' *Settings* tab in the *Config Vars* section:
+
+![Heroku Configuration](./../../../assets/heroku_config_vars.png)
+
+Next, you have to add the Heroku git remote to your repository remotes. shopware-pwa cli already created a Git repository inside your project, so you just have to run
+
+```bash
+heroku git:remote -a my-pwa-project
+```
+
+where `my-pwa-project` is the name of your Heroku app.
+
+## Node and package manager versions
+
+Next, we have to define a node environment for our Heroku app. We can do that by adding an `engines` config to our `packages.json` file. For me, using the same versions that I use for local development worked just fine.
+
+```json
+{
+  "name": "shopware-pwa-project",
+  "version": "1.0.0",
+  "engines": {
+    "node": "10.18.x",
```

Comment:
could we advise to use node 12.x? As it is a current LTS and 10.x comes to maintenance stage in a few days
https://nodejs.org/en/about/releases/

Reply:
I haven't tested that, but will include it :)

Reply:
Will we need to give a specific yarn version for that to work? (Just want to have a working combination of node and yarn in the docs)